src/components/ha-service-control.ts

```diff
@@ -387,6 +404,14 @@ export class HaServiceControl extends LitElement {
     });
   }
 
+  private async _fetchManifest(integration: string) {
+    try {
+      this._manifest = await fetchIntegrationManifest(this.hass, integration);
+    } catch (err) {
+      // Ignore if loading manifest fails. Probably bad JSON in manifest
```

Comment:
It will not clear the old manifest now, before fetching a new one, set `this._manifest = undefined`